CSS invert function (#41147)
mdn/content: files/en-us/web/css/filter-function/invert/index.md

## Syntax

```css
/* No inversion */
invert(0)

/* Complete inversion */
invert()
invert(1)
invert(100%)

/* 60% inversion */
invert(.6)
invert(60%)
```

### Parameters

> The `invert( <amount> )` function's parameter is:

Reviewer: ```suggestion

```

> - `amount` {{Optional_Inline}}

Reviewer: I think, to be closer to the Formal syntax, we should not use "amount", but instead:
```suggestion
- {{cssxref("&lt;number&gt;")}} or {{cssxref("&lt;percentage&gt;")}} {{Optional_Inline}}
```

>   - : The amount of the conversion, specified as a {{cssxref("&lt;number&gt;")}} or a {{cssxref("&lt;percentage&gt;")}}. A value of `100%` is completely inverted, while a value of `0%` leaves the input unchanged. Values between `0%` and `100%` are linear multipliers on the effect. The initial value for {{Glossary("interpolation")}} is `0`. The default value is `1`.

Reviewer: And then the description could read:
```suggestion
  - : Specifies the amount of the conversion. A value of `100%` is completely inverted, while a value of `0%` leaves the input unchanged. Values between `0%` and `100%` are linear multipliers on the effect. The initial value for {{Glossary("interpolation")}} is `0`. The default value is `1`.
```

## Formal syntax

{{CSSSyntax}}

> ## SVG filter

Reviewer: all the same comments for the `blur()` function page PR (https://github.com/mdn/content/pull/41006) apply here as well, including the comments in the "Examples" section

Reviewer: since this is outside the scope of this PR, let's take it up in a follow up
